docs/develop/power-automate-integration.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: 'Run Office Scripts with Power Automate'
 description: 'How to get Office Scripts for Excel on the web working with a Power Automate workflow.'
-ms.date: 11/01/2021
+ms.date: 03/08/2022
```

Comment:
More of an overall comment, I think the parameters and returns docs could be made more useful by tying each of the features to what they would do in Power Automate. eg default parameters will prefill the Power Automate parameter field with the value, union types will create a parameter with a drop down of those types, optional parameters will make the parameter optional in Power Automate, etc